packages/app/src/locales/en.json

```diff
@@ -721,5 +721,9 @@
                 "callData": "calldata"
             }
         }
+    },
+    "systemAlert": {
+        "indexerDelayed": "Transaction indexing is {indexerDelayInHours} hours behind. Transactions are being processed normally and will gradually show up. You can also use other explorers like <a href=\"https://hyperscan.xyz/\">hyperscan.xyz</a> meanwhile.",
```

Comment:
I'm concerned about the wording here. Users may misunderstood that _Indexing is N hours behind_ means that only in N hours they will see the data they have just submitted. But in fact, it's a delay at the moment, but the Explorer may catch up much quicker.

Not sure what is the better option, I'm just thinking out loud.

Reply:
Agree, but I think it's more or less good to start with. I think showing number of blocks behind might be confusing.